code/modules/goonchat/browserOutput.css

```diff
@@ -322,7 +321,7 @@ h1.alert, h2.alert		{color: #99aab5;}
 .round_setup			{color: #db0000;		font-family: impact; font-size: 1.25em;}
 .round_header			{color: #db0000; 	display: block; text-align: center; font-family: courier; font-weight: bold; font-size: 2em;}
 .round_body				{color: #6685f5; 	display: block; text-align: center; font-family: trebuchet-ms; font-weight: bold; font-size: 1.5em;}
-.event_announcement		{color: #600d48; 	font-family: arial-narrow; font-weight: bold; font-size: 1.5em;}
```

Comment:
Just to be clear its intentional to remove the `font-weight`?
I'd suspect you want that for an announcement unless its wrapped elsewhere.

Reply:
It is. This wasn't used anywhere, and the place where it could fit already used bold for the start of it.
The body of the announcement wasn't bold and doesn't need to be with this.